minijanus.js

```diff
@@ -89,8 +89,19 @@ JanusSession.prototype.create = function() {
 /** Destroys this session. **/
 JanusSession.prototype.destroy = function() {
   // Note that this message, unlike others, does *not* receive a response from janus, so there is no promise returned.
-  this.send("destroy");
+  try {
+    this.send("destroy").catch((e) => {});
+  } catch (e) {}
```

Comment:
And the empty catch clause here? Seems like there should be eitehr a rejection handler or an await with a catch, and they should at least log something. If you are expecting it to fail because the socket is disconnected then you should just check for that state and not actually try and send destory to the server.

Reply:
I think I can remove the try/catch block, but the empty catch handler on the promise should stay there. The transaction will be sent and then the promise will be immediately rejected. This is fine because we don't expect a response for the destroy message, but we don't want an unhandled promise exception. So I think the catch handler should stay.